### PR TITLE
Bug 2040535: Added support for customized wizard

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

**Fixes**: 
https://bugzilla.redhat.com/show_bug.cgi?id=2040535

**Analysis / Root cause**: 
Customized wizard wasnt searching for datasources.

**Solution Description**: 
The customized wizard will look for datasources for baseimage

After:
image

Before:
image

### CODE REVIEW

**Comment by avivtur** [comment]
/retitle Bug 2040535: Added support for customized wizard

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx [hunk 1]
```diff
@@ -406,6 +411,8 @@ const CreateVMWizard = connect(wizardStateToProps, wizardDispatchToProps, null,
     }),
 })(CreateVMWizardComponent);
 
+const imagesNamespace = isUpstream() ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS;
```
Comment (avivtur) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx:414]: suggestion: this already exists as `getOSImagesNS` in src/utils/common.ts

Comment (metalice) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx]: done :)

**Pull request merged**